app/components/katalyst/table_component.rb

```diff
@@ -85,6 +89,10 @@ def sorting
       collection.sorting if collection.respond_to?(:sorting)
     end
 
+    def generate_ids?
+      @generate_ids
```

Comment:
is it `!!@generate_ids`?

Reply:
or `@generate_ids.present?`? 

https://stackoverflow.com/questions/28498342/converting-anything-to-a-boolean-in-ruby

Reply:
I've gone with `@generate_ids.present?` to match the others in the class